src/components/single-create.ts

```diff
@@ -130,9 +155,16 @@ export default class SingleCreate {
         const coordiante = map.getCoordinateFromPixel([x - mapElement.offsetLeft, y - mapElement.offsetTop]);
         const geomPoint = new Point(coordiante);
         const feature = new Feature({
-          geometry: geomPoint,
+          geom: geomPoint,
+          id: Number(`${Math.round(coordiante[0])}${Math.round(coordiante[1])}`),
+          isSelected: true
         });
-        useStore().setSelectedFeature(feature)
+        feature.setGeometryName('geom');
+        if (useStore().getMaxElement() === 1) {
+          useStore().removeSelectedFeature(useStore().getCurrentItemId(), 'id');
+        }
+        useStore().setCurrentItemId(feature.get('id'))
+        useStore().addSelectedFeature(feature)
```

Comment:
may be at some point would be good to refactor with a class variable `store = useStore()` for readability